ciao-controller/command.go

```diff
@@ -106,12 +106,25 @@ func (c *controller) confirmTenantRaw(tenantID string) error {
 	}
 
 	if tenant == nil {
-		tenant, err = c.ds.AddTenant(tenantID)
+		_, err = c.ds.AddTenant(tenantID)
 		if err != nil {
 			return err
 		}
 	}
 
+	return nil
+}
+
+func (c *controller) confirmCNCI(tenantID string) error {
```

Comment:
starting the CNCI only when starting instances was a design choice. How do people feel about this? I personally don't believe the optimization of starting the CNCI whenever there's any activity at all is worth the complexity it adds.